Soomla/Assets/Plugins/Soomla/Profile/SoomlaProfile.cs

```diff
@@ -68,18 +71,27 @@ public class SoomlaProfile
 
 #if SOOMLA_FACEBOOK
 			unreadyProviders++;
-			providers.Add(Provider.FACEBOOK, new FBSocialProvider());
 #endif
 #if SOOMLA_GOOGLE
 			unreadyProviders++;
-			providers.Add(Provider.GOOGLE, new GPSocialProvider());
 #endif
 #if SOOMLA_TWITTER
 			unreadyProviders++;
-			providers.Add(Provider.TWITTER, new TwitterSocialProvider());
 #endif
 #if SOOMLA_GAMECENTER
 			unreadyProviders++;
+#endif
+
+#if SOOMLA_FACEBOOK
+			providers.Add(Provider.FACEBOOK, new FBSocialProvider());
+#endif
+#if SOOMLA_GOOGLE
+			providers.Add(Provider.GOOGLE, new GPSocialProvider());
+#endif
+#if SOOMLA_TWITTER
+			providers.Add(Provider.TWITTER, new TwitterSocialProvider());
+#endif
+#if SOOMLA_GAMECENTER
 			providers.Add(Provider.GAME_CENTER, new GameCenterSocialProvider());
 #endif
```

Comment:
Code above deny to handle false invocations (native providers send `.providerBecameReady()` message immediately in their constructors, so if we'll use only native providers, `OnProfileInitialized` will be invoked <NUMBER_OF_PROVIDERS> times.